pytorch/inference/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -177,10 +177,15 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
  && pip install --no-cache-dir -U ${PT_INFERENCE_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
 
 RUN pip uninstall -y model-archiver multi-model-server \
- && pip install captum \
  && pip install torchserve==${TS_VERSION} \
  && pip install torch-model-archiver==${TS_VERSION}
 
+# Install TorchServe pypi dependencies directly from their requirements.txt file
+# NOTE: This also brings in unnecessary cpu dependencies like nvgpu
```

Comment:
note, we are installing based on TS requirements, but nvgpu will be brought into the cpu image